flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamOverAggregate.scala

```diff
@@ -112,7 +113,12 @@ class DataStreamOverAggregate(
               "condition.")
         }
       case _: RowTimeType =>
-        throw new TableException("OVER Window of the EventTime type is not currently supported.")
+        if (overWindow.lowerBound.isUnbounded && overWindow.upperBound.isCurrentRow) {
+          createUnboundedAndCurrentRowEventTimeOverWindow(inputDS)
```

Comment:
No problem. So,Can I change this JIRA.'s title, And open a new JIRA. to addresses the `RANGE` case ? @fhueske 
Thanks,
SunJincheng